packages/fast-tooling-react/src/css-editor/color/color.props.ts

```diff
@@ -1,22 +1,18 @@
 import { ManagedClasses } from "@microsoft/fast-jss-manager-react";
 import { CSSColorClassNameContract } from "./color.style";
+import { CommonControlConfig, ControlOnChangeConfig } from "src/form/templates";
```

Comment:
this will fail since there is no `src` folder in the distributed packages, use a relative link.